### DISCUSSION TITLE
Why did they start using the full ORM? (EF Core)

### DISCUSSION THREAD

**Discussion by itprodavets** [dotnet/eShop#461] [General]
I previously saw in your project the use of direct queries to the database. How much more productive is EF Core now than direct queries to the database?

**Comment by bytefish** [comment]
It has been refactored from Dapper to EF Core by now:

* https://github.com/dotnet/eShop/blob/main/src/Ordering.API/Application/Queries/OrderQueries.cs

It's confusing, that Dapper is still a PackageReference, because it's nowhere to be used.

Anyway, don't use this repository as guidance on how to use EF Core. The EF Core team says:

> No, in general this repo should not be used for guidance on how to use EF Core. There are lot of things in this code that the team considers anti-patterns. ([See Here](https://github.com/dotnet/eShop/issues/302#issuecomment-2053585787))
